--- PKGBUILD.0.8.2-2.orig	2025-12-16 15:20:09.761894630 +0100
+++ PKGBUILD	2025-12-16 15:20:29.315392387 +0100
@@ -7,13 +7,12 @@
 pkgver=0.8.2
 pkgrel=2
 pkgdesc="A Vulkan overlay layer for monitoring FPS, temperatures, CPU/GPU load and more."
-arch=('x86_64')
+arch=('x86_64' 'aarch64')
 url="https://github.com/flightlessmango/MangoHud"
 license=('MIT')
-makedepends=('appstream' 'cmocka' 'git' 'glslang' 'libxnvctrl' 'libxrandr' 'meson' 'python-mako' 'vulkan-headers')
+makedepends=('appstream' 'cmocka' 'git' 'glslang' 'libxrandr' 'meson' 'python-mako' 'vulkan-headers')
 depends=('dbus' 'fmt' 'gcc-libs' 'glfw' 'hicolor-icon-theme' 'libglvnd' 'libx11' 'libxkbcommon' 'python' 'python-matplotlib' 'python-numpy' 'vulkan-icd-loader' 'wayland')
-optdepends=('gamescope: Use MangoApp as an overlay within gamescope'
-            'libxnvctrl: NVIDIA GPU stats by XNVCtrl')
+optdepends=('gamescope: Use MangoApp as an overlay within gamescope')
 replaces=("$pkgname-common" "$pkgname-common-wayland" "$pkgname-common-x11" "mangoapp" "$pkgname-x11" "$pkgname-wayland")
 conflicts=("$pkgname-common" "$pkgname-common-wayland" "$pkgname-common-x11" "mangoapp" "$pkgname-x11" "$pkgname-wayland")
 source=("$pkgname-$pkgver.tar.xz::https://github.com/flightlessmango/MangoHud/releases/download/v$pkgver/$_pkgname-v$pkgver-Source.tar.xz")
@@ -30,7 +29,8 @@
 build() {
     arch-meson "$_pkgname-v$pkgver" build \
         -Dmangoapp=true \
-        -Dmangohudctl=true
+        -Dmangohudctl=true \
+        -Dwith_xnvctrl=disabled
 
     meson compile -C build
 }
